client-java/instrumentation-shared/src/main/java/org/evomaster/client/java/instrumentation/shared/SecuritySharedUtils.java

```diff
@@ -0,0 +1,9 @@
+package org.evomaster.client.java.instrumentation.shared;
+
+public class SecuritySharedUtils {
+
+    /**
+     * Reserved IP address for SSRF verifier.
+     */
+    public static final String HTTP_CALLBACK_VERIFIER = "127.0.0.4";
```

Comment:
as we are controlling the port (which we could make ephemeral) do we need a different localhost? especially considering it would not work out-of-the-box for Mac, isn't it?

Reply:
Yes, for macOS it won't work. Initially I thought we may expand this in future so allocated a new IP. However, it makes sense to run it in `localhost`. I'll revert the changes.